syncer/ghost.go

```diff
@@ -127,16 +126,6 @@ func (g *Ghost) Apply(tables []*filter.Table, statement string, stmt ast.StmtNod
 	return nil, schema, table, nil
 }
 
-// InOnlineDDL implements interface
-func (g *Ghost) InOnlineDDL(schema, table string) bool {
```

Comment:
@GregoryIan what's your opinion about removing these unused interfaces.

Reply:
It doesn't matter, add it back when you need it.